c2rust-analyze/src/util.rs

```diff
@@ -170,28 +188,34 @@ pub fn ty_callee<'tcx>(tcx: TyCtxt<'tcx>, ty: Ty<'tcx>) -> Callee<'tcx> {
     };
 
     match *ty.kind() {
-        ty::FnDef(did, substs) => {
+        ty::FnDef(def_id, substs) => {
```

Comment:
nit: `did` is used elsewhere and I believe is common in `rustc` code. It might be beneficial to keep things consistent

Reply:
rustc itself is not especially consistent in this regard, though it does seem to favor `def_id` over `did` somewhat.

Reply:
> rustc itself is not especially consistent in this regard, though it does seem to favor `def_id` over `did` somewhat.

Yeah, I've noticed that as well.  `def_id` is slightly more preferred I think.  Changing it here got rid of the `def_id: did` in a `struct` initializer, so it seemed slightly cleaner.